src/ngraph/pass/algebraic_simplification.hpp

```diff
@@ -32,7 +33,9 @@ class ngraph::pass::AlgebraicSimplification : public FunctionPass
     AlgebraicSimplification()
         : FunctionPass()
     {
+        auto property = pass::PassPropertyMask(pass::PassProperty::REGULAR_FUSIONS) |
```

Comment:
There's a list initializer constructor for EnumMask, so you could also do it this way: `pass::PassPropertyMask property{pass::PassProperty::REGULAR_FUSIONS, pass::PassProperty::REQUIRE_STATIC_SHAPE}`